packages/app/src/flame/Flam3.tsx

```diff
@@ -218,6 +218,10 @@ export function Flam3(props: Flam3Props) {
     })
 
     createEffect(() => {
+      const _redrawWhenChanged = [
```

Comment:
This is supposed to be used later or? Can we add a comment for this or explain why and for what?

Reply:
I split this into a separate effect and added a comment